### PR TITLE
move sharetools

### PR DESCRIPTION
**Pull request by MelizzaP**
remove share tools from bulbs-video-meta

### CODE REVIEW

**Comment by MelizzaP** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;NaGsT6NZ&#x2F;14-avc-homepage-player"> AVC homepage player</a>

**Comment by kand** [comment]
👍

File: bulbs/styles/static/styles/bulbs-styles/mixins/multi-line-truncate.scss [hunk 1]
```diff
@@ -10,12 +10,12 @@
   overflow: hidden;
   position: relative;
   &:after {
-    background: linear-gradient(to right, rgba(255, 255, 255, 0), $bg-color 50%);
+    background: linear-gradient(to right, $bg-color, $bg-color 50%);
```
Comment (jmelvnsn) [inline bulbs/styles/static/styles/bulbs-styles/mixins/multi-line-truncate.scss]: slick

**Comment by jmelvnsn** [comment]
👍

**Pull request merged**